### PR TITLE
chore: use cypress for e2e testing

### LINKED ISSUE #40: Refactor contract_test to use cypress

### PR DESCRIPTION
**Pull request by stigbd**
Resolves #40

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -88,19 +90,56 @@ def contract_tests(session: Session) -> None:
...
+        with session.chdir("e2e"):
+            # Runs in "e2e"
+            session.run(
+                "npm",
+                "install",
+                external=True,
+            )
+            session.run(
+                "npx",
+                "cypress",
```
Comment (jeffreiffers) [inline noxfile.py:122]: Ønsker du ikke at vi skal record og lagre resultatene i cypress dashboard?

**Review by valosnah — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -101,6 +101,30 @@ The static files to be served should be store under `/srv/www/static-rdf-server/
...
+```Shell
+% pipx install nox
+% pipx install poetry
+% pipx inject nox nox-poetry
+```
+
+### Install software
+
+```Shell
+% git clone https://github.com/Informasjonsforvaltning/dataservice-publisher.git
```
Comment (tenstad) [inline README.md:121]: Ka farsken skal man med dataservice publisher?

File: e2e/cypress/support/e2e.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```
Comment (tenstad) [inline e2e/cypress/support/e2e.js:20]: Fikser du newlines nederst i filene?

File: e2e/cypress/e2e/ontology.cy.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
+/// <reference types="cypress" />
+
+// Welcome to Cypress!
+//
+// This spec file contains a variety of sample tests
+// for a todo list app that are designed to demonstrate
+// the power of writing tests in Cypress.
+//
```
Comment (tenstad) [inline e2e/cypress/e2e/ontology.cy.js:8]:
```suggestion
```

File: e2e/cypress/support/e2e.js (hunk 3 above)
Comment (stigbd) [reply e2e/cypress/support/e2e.js]: Fiksa i 8711065c5413bf8ee5b37be0a81e134cdd565a4f

File: README.md (hunk 2 above)
Comment (stigbd) [reply README.md]: Fiksa i 7a4a83c38e6bf1d2aaf25495be5b237a969ebb7a

File: e2e/cypress/e2e/ontology.cy.js (hunk 4 above)
Comment (stigbd) [reply e2e/cypress/e2e/ontology.cy.js]: Fiksa i b64ee16d4552acb91da73c79ed36c8d789ab9bee

**Review by tenstad — APPROVED** [review]

**Review by jeffreiffers — DISMISSED** [review]

**Pull request merged**